src/main/kotlin/model/ContractInternal.kt

```diff
@@ -26,7 +26,7 @@ data class ContractInternal(
     override val monthlySalary: Double,
     override val hoursPerWeek: Int,
 
-    val holidayHours: Int,
+    val leaveHours: Int,
```

Comment:
Deze mag ook holidayHours blijven. Dit is het aantal uur dat iemand krijgt vanuit zijn contract